### PR TITLE
i want to believe

### PR DESCRIPTION
**Pull request by Iamgoofball**

### CODE REVIEW

File: code/game/gamemodes/abduction/abduction.dm [hunk 1]
```diff
@@ -139,7 +139,6 @@
 	agent_landmarks.len = max_teams
 	scientist_landmarks.len = max_teams
 	for(var/obj/effect/landmark/abductor/A in landmarks_list)
-		world << "Found this [A]"
```
Comment (optimumtact) [inline code/game/gamemodes/abduction/abduction.dm]: RIP

Comment (Iamgoofball) [inline code/game/gamemodes/abduction/abduction.dm]: chem trails spread the gubmint mind controls

Comment (Incoming5643) [inline code/game/gamemodes/abduction/abduction.dm]: Found this.

Ayyyyyyyyyyy

**Comment by AnturK** [comment]
Welp.

**Comment by Chocobro** [comment]
what is this also cool 24 hours

**Comment by phil235** [comment]
It's a one line fix that remove a debug message that's broadcasted to everyone, which means people can deduce the gamemode from it. It's thus fairly urgent.

**Pull request merged**